omnibus/config/templates/datadog-agent/systemd.service.erb

```diff
@@ -1,14 +1,15 @@
 [Unit]
 Description="Datadog Agent"
 After=network.target
+Wants=datadog-agent-trace.service datadog-agent-process.service
```

Comment:
We already have `BindsTo=` to the other units (`datadog-agent-trace.service datadog-agent-process.service`) so I'm not sure Wants is needed here.

Reply:
actually we do. Otherwise starting `datadog-agent.service` will _not_ start `datadog-agent-trace.service` and `datadog-agent-process.service`.

Reply:
Correct, the relationship does not go both ways.